package.json

```diff
@@ -52,8 +53,8 @@
     "semantic-release": "15.13.12",
     "travis-deploy-once": "5.0.11"
   },
-  "dependencies": {
-    "@testing-library/dom": "5.1.1"
+  "peerDependencies": {
+    "@testing-library/dom": "^5.1.1"
```

Comment:
```suggestion
    "@testing-library/dom": ">=5"
```

Reply:
Peer Dependencies should have a version range that's as permissible as possible. That way people can install their own version without conflicting with the one this package says it needs.

Reply:
@kentcdodds what if `@testing-library/dom` releases a v6 that has breaking changes that affect this package? That's why I went with a caret range. Maybe the caret range should be `^5.0.0` though?

Reply:
Chances are that a breaking change in `@testing-library/dom` wont impact this package. If it does, then we can fix this. The last thing I want is to add to the already ridiculous amount of warnings people get when they install deps in their project.